src/test/kotlin/org/stellar/walletsdk/WalletTest.kt

```diff
@@ -1,25 +1,59 @@
 package org.stellar.walletsdk
 
 import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
 import kotlin.test.assertTrue
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
-import org.stellar.sdk.Network
 
-class WalletTest {
-  private val wallet = Wallet("https://horizon-testnet.stellar.org", Network.TESTNET.toString())
+internal class WalletTest {
+  private val wallet = Wallet(Constants.HORIZON_URL, Constants.NETWORK_PASSPHRASE)
 
-  @Test
-  fun `create() generates Stellar public and secret keys`() {
-    val accountKeys = wallet.create()
-    val publicKey = accountKeys.publicKey
-    val secretKey = accountKeys.secretKey
+  @Nested
+  @DisplayName("create")
+  inner class Create {
+    @Test
+    fun `generates Stellar public and secret keys`() {
+      val accountKeys = wallet.create()
+      val publicKey = accountKeys.publicKey
+      val secretKey = accountKeys.secretKey
 
-    // Public key
-    assertEquals(56, publicKey.length)
-    assertTrue(publicKey.startsWith("G"))
+      // Public key
+      assertEquals(56, publicKey.length)
+      assertTrue(publicKey.startsWith("G"))
 
-    // Secret key
-    assertEquals(56, secretKey.length)
-    assertTrue(secretKey.startsWith("S"))
+      // Secret key
+      assertEquals(56, secretKey.length)
+      assertTrue(secretKey.startsWith("S"))
+    }
+  }
+
+  @Nested
+  @DisplayName("fund")
+  inner class Fund {
```

Comment:
I'd like to add more tests that check the operations in transactions but I didn't find an easy way to do it right now. For example, it would be nice to check if the starting balance for a sponsored fund transaction is `0`. We'll spend more time on tests later.